### DISCUSSION TITLE
23.12.13 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#88] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 개발한 기능 정리
- 내일은 어떤 일을 하나요?
    - 발표 준비
    - 다함께 리드미 작성

**Reply by minju412** [reply]
ERD 작성하기

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 문서화
  - 구현 기능은 지금까지 작성한 PR이나 Issue 내용으로 사용
  - 작업 과정을 알 수 있는 시퀀스 다이어그램을 작성 중
- 내일은 어떤 일을 하나요?
  - 발표 준비
  - 리드미 작성 준비
